src/TestInfrastructure/VsObjectModelMocks/ProjectItemsMock.cs

```diff
@@ -131,24 +142,24 @@ IEnumerator ProjectItems.GetEnumerator()
 
         ProjectItem ProjectItems.Item(object index)
         {
-            return this.items[(int)index + 1];
+            return this.items[(int)index - 1]; // VS item indexing starts at 1
```

Comment:
Miscellaneous: index wasn't calculated correctly (which suggests this method isn't actually called) by the tests.